packages/movit/build.sh

```diff
@@ -0,0 +1,16 @@
+TERMUX_PKG_HOMEPAGE=https://movit.sesse.net/
+TERMUX_PKG_DESCRIPTION="The modern video toolkit. "
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_LICENSE="GPL-2.0"
+TERMUX_PKG_VERSION=1.7.1
+TERMUX_PKG_SRCURL=https://movit.sesse.net/movit-${TERMUX_PKG_VERSION}.tar.gz
+TERMUX_PKG_SHA256=b33073b705f0ccb6ac4942cf51151515407b40bb4e9a2dd0228c1c2cb1fbc11a
+TERMUX_PKG_DEPENDS="fftw, libepoxy"
+TERMUX_PKG_BUILD_DEPENDS="eigen, googletest, sdl2"
+TERMUX_PKG_SUGGESTS="$TERMUX_PKG_BUILD_DEPENDS"
+TERMUX_PKG_BUILD_IN_SRC=true
+TERMUX_PKG_AUTO_UPDATE=true
+# build strategy used by Fedora
+TERMUX_PKG_EXTRA_CONFIGURE_ARGS="--disable-static"
+TERMUX_PKG_EXTRA_MAKE_ARGS="TESTS="
+TERMUX_PKG_BLACKLISTED_ARCHES="arm"
```

Comment:
Is the build generally broken on 32bit ARM, or what is the hangup with building it for that architecture?

Reply:
Below TUR PR Actions prompt error when build 32-bit ARM package (https://github.com/termux-user-repository/tur/actions/runs/9288882338/job/25561402931?pr=1054) when scanning and packaging files. 

Not sure if it is related to build test being disabled (following the practice of [Fedora build script](https://src.fedoraproject.org/rpms/movit/blob/epel8/f/movit.spec)). Archlinux build script relies on more [complete `gtest` installation](https://gitlab.archlinux.org/archlinux/packaging/packages/gtest/-/blob/main/PKGBUILD?ref_type=heads). 

I am not sure, shall I use `TERMUX_PKG_UNDEF_SYMBOLS_FILES` to suppress the error?

```
install -m 0644 effect_chain.h effect_util.h effect.h input.h image_format.h init.h util.h defs.h resource_pool.h fp16.h ycbcr.h version.h flat_input.h ycbcr_input.h ycbcr_422interleaved_input.h lift_gamma_gain_effect.h white_balance_effect.h gamma_expansion_effect.h gamma_compression_effect.h colorspace_conversion_effect.h alpha_multiplication_effect.h alpha_division_effect.h saturation_effect.h blur_effect.h diffusion_effect.h glow_effect.h unsharp_mask_effect.h mix_effect.h overlay_effect.h padding_effect.h resample_effect.h dither_effect.h deconvolution_sharpen_effect.h fft_pass_effect.h vignette_effect.h slice_effect.h complex_modulate_effect.h luma_mix_effect.h fft_convolution_effect.h ycbcr_conversion_effect.h deinterlace_effect.h sandbox_effect.h mirror_effect.h resize_effect.h multiply_effect.h fft_input.h /data/data/com.termux/files/usr/include/movit/
mkdir -p /data/data/com.termux/files/usr/lib/pkgconfig/
install -m 644 movit.pc /data/data/com.termux/files/usr/lib/pkgconfig/
INFO: READELF=llvm-readelf ... /data/data/com.termux/files/usr/bin/llvm-readelf
INFO: Generating symbols regex to /data/data/com.termux/files/usr/tmp/tmp.SOfu4q9HhO
INFO: Done ... 0s
INFO: Total symbols 2136
INFO: Identifying files with nproc=2
INFO: Done ... 1s
INFO: Found 1 / 49 files
INFO: Running symbol checks on 1 files with nproc=2
INFO: Done ... 0s
INFO: Found files with undefined symbols
INFO: Showing result
ERROR: ./lib/libmovit.so contains undefined symbols:
    69: 00000000     0 NOTYPE  GLOBAL DEFAULT   UND __aeabi_uidiv
   120: 00000000     0 NOTYPE  GLOBAL DEFAULT   UND __aeabi_idiv
   155: 00000000     0 NOTYPE  GLOBAL DEFAULT   UND __aeabi_ul2d
   205: 00000000     0 NOTYPE  GLOBAL DEFAULT   UND __aeabi_uidivmod
   217: 00000000     0 NOTYPE  GLOBAL DEFAULT   UND __aeabi_idivmod
INFO: Done ... 1s
ERROR: Refer above
INFO: Found 1 files with undefined symbols after exclusion
Error: Process completed with exit code 1.
```

Reply:
So it builds, but the symbol checker complains.